code/index.html

```diff
@@ -1,22 +1,166 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link href="https://fonts.googleapis.com/css?family=Poppins|Roboto&display=swap" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="https://kit.fontawesome.com/41a2006263.js" crossorigin="anonymous"></script>
+
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+
+  <!-- Some kind of logo and navbar -->
+
+  <header>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+
+    <nav>
+      <a href="">Home</a>
+      <a href="#food">Food</a>
+      <img class="logo" src="wine.svg" alt="logo">
+      <a href="#wines">Wines</a>
+      <a href="#footer">Contact</a>
+    </nav>
+
+    <div>
+      <img class="logosmall" src="wine.svg" alt="logo">
+    </div>
+
+    <div class="name">
+      <p class="name">TableTop</p>
+    </div>
+  </header>
+
+
+  <!-- The big news content -->
+
+  <section class="big-news-container">
+    <div class="big-news">
+      <img src="images/insta_citron_skumpa_mork_haller.jpg" class="big-news-picture" alt="Champagne image">
```

Comment:
A screen reader for example will read "image champagne" so there's no need to add "image" to an alt text
```suggestion
      <img src="images/insta_citron_skumpa_mork_haller.jpg" class="big-news-picture" alt="Champagne">
```